app/shared/productTypes.tsx

```diff
@@ -1,25 +1,43 @@
 import React from "react";
-import { NoMembership } from "../client/components/cancel/membership/noMembership";
-import { MeValidator } from "../client/components/cancellationFlowWrapper";
+import { MeValidator } from "../client/components/cancel/cancellationFlowWrapper";
+import { NoMembership } from "../client/components/noMembership";
 import { MeResponse } from "./meResponse";
 
-export type ProductName = "membership" | "recurring contribution";
+export type ProductFriendlyName = "membership" | "recurring contribution";
```

Comment:
Customers might not be thinking of their monthly or annual contribution as a 'recurring contribution'. Maybe raise another ticket to make it contextual to the billing period, then you can use 'monthly' or 'annual'.

Reply:
will add a TODO in the codebase to cover this